pkg/gatewayserver/io/udp/udp.go

```diff
@@ -169,13 +170,16 @@ func (s *srv) handlePacket(ctx context.Context, packet encoding.Packet) {
 		}
 	}
 
-	if s.firewall != nil {
-		if err := s.firewall.Filter(packet); err != nil {
-			if !errors.IsResourceExhausted(err) || ratelimit.Require(s.limitLogs, ratelimit.NewCustomResource(eui.String())) == nil {
-				logger.WithError(err).Warn("Packet filtered")
-			}
+	if err := s.firewall.Filter(packet); err != nil {
+		if !errors.IsResourceExhausted(err) {
+			goto filtered
+		}
+		if ratelimit.Require(s.limitLogs, ratelimit.NewCustomResource(eui.String())) != nil {
 			return
 		}
```

Comment:
So we silently drop rate-limited packets?

Reply:
Yes, specifically we avoid printing the rate limiting logs _from the same gateway_. This was the old behavior too, but the line was so long that it triggered linting warnings.